.github/workflows/check-outdated-trusted-root.yml

```diff
@@ -7,6 +7,9 @@ on:
 jobs:
   check:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      issues: write
```

Comment:
Please also add a top-level `permissions` section. For consistency with the other workflows and also because it is a little "safer" if additional jobs are added after-the-fact. The top-level `permissions` could just be `contents: read`, the more specific job-level permissions will override this.

Reply:
See #127 for further context (that's where I initially introduced the permissions).